docs/guides/coding-conventions/components.md

```diff
@@ -28,6 +28,29 @@ The organization of a Svelte component **must** start with the script, followed
 <style></style>
 ```
 
+## Props
+
+Each Svelte component can contain an arbitrary number of "prop" variables, useful for passing data between components easily without the direct use of stores.
+
+When initializing props, we use the following conventions:
+
+- Every prop **MUST** be typed
+- Props without a default value should not be initialized at all, resulting in an `undefined` value
+
+**Bad**
+
+```typescript
+export let myPropObject: IPropObject = {} // this will be truthy in conditionals
+export let showSomething: boolean = false
+```
+
+**Good**
+
+```typescript
+export let myPropObject: IPropObject
+export let showSomething: boolean
+```
+
 ## Reactivity
```

Comment:
Excellent!

Reply:
![image](https://user-images.githubusercontent.com/45079109/187897766-978a227f-5684-4697-8666-b2dde015577f.png)

Reply:
Removed it :sweat_smile: